bindings/node/binding.cc

```diff
@@ -0,0 +1,28 @@
+#include "tree_sitter/parser.h"
```

Comment:
Do I need to make adjustments to the build & deploy pipeline to adjust to the bindings now living here?